docs/current/data-model-howto.md

```diff
@@ -50,6 +50,8 @@ Your data center is expressed as an inventory of _hosts_. Each host belongs to a
 
 *Cluster*: A grouping of hosts under a specific site.
 
+*Configuration*: The general name given for the collection of `Operating System`, `Assets`, `Boot` and `Workflows`. Please refer below for explanation.
```

Comment:
What is the difference between 'configuration' and 'source of truth' which appears to be widely used in these docs?

Reply:
Configuration is the collection of OS, Assets, Boot and Workflows. These need to coexist but not necessarily at the same location as the sites directory, as we have in line vs split configuration options. SOT refers to the set that contains the configuration and the sites.

Reply:
I think a diagram could be helpful to kind of visually show the differences between the two configuration paths. I'll take a crack at that this morning and have something we can talk to